.github/workflows/tests.yaml

```diff
@@ -3,11 +3,12 @@ name: Tests
 
 on:
   push:
-    branches:
-      - 'main'
+    branches: ['main']
```

Comment:
just out of curiosity, why this change?

Reply:
In my eyes it's either:
```
  pull_request:
    branches:
      - '*'
  merge_group:
    types:
      - checks_requested
```

or the option I opted for so that all the lists are the same format.